<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #22222e
</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1E90FF</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/BankLogo.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="reg_text">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Калькулятор кредита</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: CornflowerBlue</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="insurance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white</string>
     </property>
     <property name="text">
      <string>Страховка</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bet">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8.4</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="reg_text_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>% ставка</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cheat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Обмануть
 кредитора?;) </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="not_cheat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Не обманывать
кредитора</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="sum_zaim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 4px solid #1E90FF;
border-radius: 30;
color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="term">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 4px solid #1E90FF;
border-radius: 30;
color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continue_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #1E90FF;
	border-radius: 30;
	color: white
}

QPushButton:pressed {
	background-color:	#6495ED;
}</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_table">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>690</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>К каждому кредиту должен быть текст
мелким шрифтом ;)</string>
    </property>
   </widget>
   <widget class="QLabel" name="table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
